### PR TITLE
Support version param in one liner script

### PR DESCRIPTION
**Pull request by youngbupark**
# Description

Support version params in one liner script

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #516 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Review by AaronCrawfis — DISMISSED** [review]
LGTM

Down the road I'm also wondering if we should link to dapr docs on installation instructions instead of duplicating it. For now it's good to duplicate.

**Comment by youngbupark** [comment]
> LGTM
> 
> Down the road I'm also wondering if we should link to dapr docs on installation instructions instead of duplicating it. For now it's good to duplicate.

+1

I validated the script but not for one liner command line. I am working on it. please wait until the validation is done.

File: install/install.ps1 [hunk 1]
```diff
@@ -3,7 +3,8 @@
 # Licensed under the MIT License.
 # ------------------------------------------------------------
 param (
-    [string]$DaprRoot = "c:\dapr"
+    [string]$DaprRoot = "/tmp/",
```
Comment (mukundansundar) [inline install/install.ps1:6]: Why is this being changed to `/tmp/`?

Comment (youngbupark) [reply install/install.ps1]: my bad. I fixed it.

**Comment by youngbupark** [comment]
for windows:
```
powershell -Command "\$script=iwr -useb https://raw.githubusercontent.com/dapr/cli/youngp/add-version-param/install/install.ps1; \$block=[ScriptBlock]::Create(\$script); invoke-command -ScriptBlock \$block -ArgumentList 1.0.0-rc.1"
```
for macos and linux,

```
curl -fsSL https://raw.githubusercontent.com/dapr/cli/youngp/add-version-param/install/install.sh | /bin/bash -s 1.0.0-rc.1

wget -q https://raw.githubusercontent.com/dapr/cli/youngp/add-version-param/install/install.sh -O - | /bin/bash -s 1.0.0-rc.1
```

**Comment by orizohar** [comment]
> for windows:
> 
> ```
> powershell -Command "\$script=iwr -useb https://raw.githubusercontent.com/dapr/cli/youngp/add-version-param/install/install.ps1; \$block=[ScriptBlock]::Create(\$script); invoke-command -ScriptBlock \$block -ArgumentList 1.0.0-rc.1"
> ```
> 
> for macos and linux,
> 
> ```
> curl -fsSL https://raw.githubusercontent.com/dapr/cli/youngp/add-version-param/install/install.sh | /bin/bash -s 1.0.0-rc.1
> 
> wget -q https://raw.githubusercontent.com/dapr/cli/youngp/add-version-param/install/install.sh -O - | /bin/bash -s 1.0.0-rc.1
> ```

Just to confirm - this should be from `master` not `youngb` right?

**Comment by youngbupark** [comment]
> > for windows:
> > ```
> > powershell -Command "\$script=iwr -useb https://raw.githubusercontent.com/dapr/cli/youngp/add-version-param/install/install.ps1; \$block=[ScriptBlock]::Create(\$script); invoke-command -ScriptBlock \$block -ArgumentList 1.0.0-rc.1"
> > ```
> > 
> > 
> > for macos and linux,
> > ```
> > curl -fsSL https://raw.githubusercontent.com/dapr/cli/youngp/add-version-param/install/install.sh | /bin/bash -s 1.0.0-rc.1
> > 
> > wget -q https://raw.githubusercontent.com/dapr/cli/youngp/add-version-param/install/install.sh -O - | /bin/bash -s 1.0.0-rc.1
> > ```
> 
> Just to confirm - this should be from `master` not `youngb` right?

Once this PR is merged to master, then we can use master :) this is for the testing before merging this PR.

**Review by orizohar — APPROVED** [review]

**Comment by youngbupark** [comment]
@orizohar Please try this. 
```
powershell -Command "$script=iwr -useb https://raw.githubusercontent.com/dapr/cli/youngp/add-version-param/install/install.ps1; $block=[ScriptBlock]::Create($script); invoke-command -ScriptBlock $block -ArgumentList 1.0.0-rc.1"
```

**Review by orizohar — APPROVED** [review]

**Review by mukundansundar — APPROVED** [review]

**Pull request merged**